plugins/modules/system/seport.py

```diff
@@ -278,7 +278,7 @@ def main():
     if not get_runtime_status(ignore_selinux_state):
         module.fail_json(msg="SELinux is disabled on this host.")
 
-    ports = module.params['ports']
+    ports = list(map(str, module.params['ports']))
```

Comment:
It might be slightly simpler as:
```
ports = [str(p) for p in ports]
```

Reply:
`elements='str'` in the argument spec is even more simple :)